tensorflow/python/ops/ragged/ragged_math_ops.py

```diff
@@ -630,6 +637,18 @@ def reduce_mean(input_tensor, axis=None, keepdims=None, name=None):
       return total / count
 
 
+def reduce_variance(input_tensor, axis=None, keepdims=False, name=None):
+  """For docs, see: _RAGGED_REDUCE_DOCSTRING."""
+  with ops.name_scope(name, 'RaggedReduceVariance', [input_tensor, axis]):
+    square_of_input = gen_ragged_math_ops.square(input_tensor)
```

Comment:
This should be `math_ops.square`, not `gen_ragged_math_ops.square`.  (Are the tests passing?  I would have expected this to be caught by the tests.)  (Similarly, line 648 below should be changed to `math_ops.square`.)

Reply:
done